app/code/Magento/Wishlist/Controller/Index/Plugin.php

```diff
@@ -30,26 +30,34 @@ class Plugin
     protected $config;
 
     /**
-     * @var \Magento\Framework\App\Response\RedirectInterface
+     * @var \Magento\Framework\Message\ManagerInterface $messageManager
```

Comment:
Looks like this change was done unintentionally.

Reply:
Done.